src/app/FakeLib/SignToolHelper.fs

```diff
@@ -65,6 +65,42 @@ let Sign (toolsPath : string) (parameters : SignParams) (filesToSign : seq<strin
 /// Signs all files in filesToSign with the certification file certFile, 
 /// protected with the password in the file passFile. 
 /// The signtool will be search in the toolPath.
```

Comment:
yes, sorry they belong to the obsolete function and I forgot moving them after inserting the new code